test/address_sanitizer_setup.sh

```diff
@@ -23,7 +23,7 @@ then
 
     ./configure
 
-    make -j
+    make -j 4
```

Comment:
nit, make -j `nproc`

Reply:
I agree that it is not so good to hard code this but what shall nproc be?

Reply:
Sorry I meant,
```
make -j `nproc`
```